packages/config/src/layer2s/honeypot.ts

```diff
@@ -45,9 +68,94 @@ export const honeypot: Layer2 = {
       }),
     ],
   },
-  riskView: UNDER_REVIEW_RISK_VIEW,
-  technology: TECHNOLOGY.UNDER_REVIEW,
-  contracts: CONTRACTS.UNDER_REVIEW,
+  riskView: makeBridgeCompatible({
+    stateValidation: RISK_VIEW.STATE_NONE,
+    dataAvailability:  {
+      ...RISK_VIEW.DATA_ON_CHAIN,
+      sources: [
+        {
+          contract: 'InputBox',
+          references: [
+            'https://etherscan.io/address/0x59b22D57D4f067708AB0c00552767405926dc768#code#F1#L30',
+          ],
+        },
+      ],
+    },
+    upgradeability: RISK_VIEW.UPGRADABLE_NO,
+    sequencerFailure: RISK_VIEW.SEQUENCER_SELF_SEQUENCE(0),
+    proposerFailure: RISK_VIEW.PROPOSER_CANNOT_WITHDRAW,
+    destinationToken: RISK_VIEW.CANONICAL,
+    validatedBy: RISK_VIEW.VALIDATED_BY_ETHEREUM,
+  }),
+  technology: {
+    stateCorrectness: {
+      name: 'Fraud proofs are in development',
```

Comment:
For `stateCorrectness` I took inspiration from Optimism's configuration file. In practice, the Honeypot does NOT have a working fraud proof system working. Development is underway to have a working fraud proof system for Cartesi Rollups. Let let me know if this description and references look ok.